drivers/audio/dmic_nrfx_pdm.c

```diff
@@ -236,8 +271,26 @@ static bool find_suitable_clock(const struct dmic_nrfx_pdm_drv_cfg *drv_cfg,
 		uint8_t         ratio_val;
 		nrf_pdm_ratio_t ratio_enum;
 	} ratios[] = {
-		{  64, NRF_PDM_RATIO_64X },
-		{  80, NRF_PDM_RATIO_80X }
+#if defined(PDM_RATIO_RATIO_Ratio32)
+		{32, NRF_PDM_RATIO_32X},
+#endif
+#if defined(PDM_RATIO_RATIO_Ratio48)
+		{48, NRF_PDM_RATIO_48X},
+#endif
+#if defined(PDM_RATIO_RATIO_Ratio50)
+		{50, NRF_PDM_RATIO_50X},
+#endif
+		{64, NRF_PDM_RATIO_64X},
+		{80, NRF_PDM_RATIO_80X},
```

Comment:
As in https://github.com/zephyrproject-rtos/zephyr/pull/79177#discussion_r1801247159, please don't remove those spaces next to braces (clang-format suggestions are not always sensible). Instead, align new entries with the existing ones.